hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/java/org/hawkular/alerts/actions/email/PluginMessageDescription.java

```diff
@@ -477,7 +476,7 @@ public String compare(CompareCondition condition) {
      * @return a description to be used on email templates
      */
     public String external(ExternalCondition condition) {
-        String description = "SystemId: " + condition.getSystemId();
+        String description = "AlerterId: " + condition.getAlerterId();
```

Comment:
Is this just a renaming of systemId -> alerterId or it is introducing a new concept.
(I guess that this will be showed below in the code, but this was on the first modified files showed in the PR).